unittests/BuildSystem/BuildSystemFrontendTest.cpp

```diff
@@ -441,11 +457,16 @@ TEST_F(BuildSystemFrontendTest, commandSkipping_NonExternalCommands_symlink) {
         outputs: ["1"]
         contents: "x"
     2:
-        tool: symlink
+        tool: shell
```

Comment:
Windows seems to be unable to rm a symlink if the target doesn't exist though it doesn't have an issue creating one.

Reply:
Can you expand on what you mean by "handle that internally to the shell tool"?

Reply:
I know it's with the deletion, but isn't there a Win32 API we can call to delete a symlink without attempting to follow it?

Reply:
I don't follow... there are Win32 APIs to do exactly that, but gnu msys rm doesn't do it properly. It says it's using lstat, but whatever implementation of lstat it's using doesn't seem to be correct.  Unless you're alluding to providing a RemoveCommand that does this properly?

Reply:
Ohhh, sorry, I completely misread the diff due to the way git is presenting it here. Disregard all this :)